src/Umbraco.Web.UI.Client/src/views/documenttypes/views/permissions/permissions.controller.js

```diff
@@ -38,13 +38,12 @@
             });
 
             contentTypeResource.getAll().then(function(contentTypes){
-
-                vm.contentTypes = contentTypes;
+                vm.contentTypes = _.where(contentTypes, {isElement: false});
 
                 // convert legacy icons
                 iconHelper.formatContentTypeIcons(vm.contentTypes);
 
-                vm.selectedChildren = contentTypeHelper.makeObjectArrayFromId($scope.model.allowedContentTypes, vm.contentTypes);
+                vm.selectedChildren = contentTypeHelper.makeObjectArrayFromId($scope.model.allowedContentTypes, contentTypes);
```

Comment:
Not sure why this is changed from `vm` to `$scope...`? Leftover from a debug session maybe?

Reply:
It's not `$scope`, `contentTypes` is the list returned from `contentTypeResource.getAll()`. 

Since `vm.contentTypes` is now a subset of `contentTypes` (the ones that aren't element types), we can't base `vm.selectedChildren` on `vm.contentTypes`, anymore, because then it wouldn't include any content types that were added before they were changed into element types. And thus you wouldn't see the actual config, only a subset of the actual config.

Reply:
Duh, I completely misread the line of code as well.. don't mind me! Looks fine, time to test! :)